docs/source/considerations.rst

```diff
@@ -111,6 +111,8 @@ agents at the same time.
    name servers.
 
 
+.. _oop:
```

Comment:
Is this necessary? I think we are not indexing to that label (even though I'm not exactly sure what do those two dots do)